include/deal.II/grid/manifold_lib.h

```diff
@@ -110,6 +110,14 @@ class PolarManifold : public ChartManifold<dim, spacedim, spacedim>
   virtual DerivativeForm<1, spacedim, spacedim>
   push_forward_gradient(const Point<spacedim> &chart_point) const override;
 
+  /**
+   * Return the (normalized) normal vector at the point @p p.
```

Comment:
Can you just `@copydoc` from the base class (or actually just say `// documentation is copied from the base class`

Reply:
This won't work -- if you use `/** ... */` then that's what doxygen is going to take. If you want doxygen to just use the base class function's documentation, then you need to use a non-doxygen comment like `// ...` here.